### PR TITLE
Fix sublist display in part of README.md

### PR DESCRIPTION
**Pull request by NathanZook**

### CODE REVIEW

**Comment by DStorck** [comment]
LGTM

**Comment by coffeepac** [comment]
Ok to test

**Comment by joejulian** [comment]
retest this please

File: README.md [hunk 1]
```diff
@@ -49,17 +49,17 @@ You will need to have the following:
 You will need the following installed on your machine:
 
 - Python 2.x (virtualenv strongly suggested)
- - pip
- - boto
- - netaddr
+  - pip
```
Comment (joejulian) [inline README.md]: Why did you change the bullet depth?

Comment (NathanZook) [inline README.md]: Because a single space indent is not recognized as starting a sublist.  If you view the file currently, verse with this change, it will be blatant.

**Comment by davidewatson** [comment]
LGTM.
